src/js/jsx/search/SearchBar.jsx

```diff
@@ -416,15 +427,19 @@ define(function (require, exports, module) {
          * Perform action based on ID
          *
          * @private
-         * @type {Datalist~onChange}
+         * @type {Datalist~onInputChange}
+         * @param {string} id of the selected option
+         * @param {string} searchTerm user input
+         * @return {{dontCloseDialog: boolean=}};
          */
-        _handleChange: function (id) {
-            if (id === null) {
+        _handleChange: function (id, searchTerm) {
+            if (!id) {
```

Comment:
Given that `id` is a non-optional parameter, this will only match if `id === ""`. Is that intentional?